src/hevm/src/EVM.hs

```diff
@@ -2552,15 +2624,14 @@ checkJump x xs = do
   self <- use (state . codeContract)
   theCodeOps <- use (env . contracts . ix self . codeOps)
   theOpIxMap <- use (env . contracts . ix self . opIxMap)
-  if x < num (len theCode) && 0x5b == (fromMaybe (error "tried to jump to symbolic code location") $ unliteral $ EVM.Symbolic.index (num x) theCode)
-    then
-      if OpJumpdest == snd (theCodeOps RegularVector.! (theOpIxMap Vector.! num x))
-      then do
-        state . stack .= xs
-        state . pc .= num x
-      else
-        vmError BadJumpDestination
-    else vmError BadJumpDestination
+  if x < num (len theCode)
+     && 0x5b == (fromMaybe (error "tried to jump to symbolic code location") $ unliteral $ EVM.Symbolic.index (num x) theCode)
+     && OpJumpdest == snd (theCodeOps RegularVector.! (theOpIxMap Vector.! num x))
+  then do
+    state . stack .= xs
+    state . pc .= num x
+  else do
+    vmError BadJumpDestination
```

Comment:
no semantic change here, just flattening the if statement since the `else` clauses are identical...